README.rst

```diff
@@ -487,18 +471,18 @@ Execution
    Encrypt or decrypt data using the AWS Encryption SDK
 
    optional arguments:
```

Comment:
Any changes to the argument helpstrings should be modified in `arg_parsing.py`, as that is where they are generated. Also, both there and in this section, back-ticks should not be used as they will be interpreted literally, not used for Sphinx formatting as they are elsewhere in this document.

Reply:
Understand, but for --version, the help string is not specified in arg_parsing.py. I think it might be related to the dummy redirect. I'm not sure about the source: frontend.py?

Reply:
Version: https://github.com/awslabs/aws-encryption-sdk-cli/blob/master/src/aws_encryption_sdk_cli/internal/arg_parsing.py#L164-L168

If we want to set a custom helpstring for the `--help` argument, we'll need to add another custom argument to that same group. I believe it would be the below, but I would need to try it verify the exact syntax for the help action:
```
version_or_action.add_argument(
    '-h',
    '--help',
    action='help',
    help='Our custom helpstring for help!'
)
parser.add_dummy_redirect_argument('--help')
```